DFD/DFD.drawio.xml

```diff
@@ -0,0 +1,2 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This and the the image should not be in the repo,its like create a branch that is a copy from upstream/main with out the dfd and then the client module